File: 1941. Check if All Characters Have Equal Number of Occurrences/solution.py
class Solution(object):
  def areOccurrencesEqual(self, s):
    """
    :type s: str
    :rtype: bool
    """
    dictionary = {}
    for letter in s:
      if letter not in dictionary:
        dictionary[letter] = 1
      else:
        dictionary[letter] += 1
    
    amount = dictionary[s[0]]
    for letter in dictionary:
      if dictionary[letter] != amount:
        return False
      
    return True

File: 1941. Check if All Characters Have Equal Number of Occurrences/solution.cpp
class Solution {
public:
  bool areOccurrencesEqual(string s) {
    unordered_map<char, int> dictionary;
    for (int index = 0; index < s.size(); index++){
      char letter = s[index];
      dictionary[letter]++;
    }

    int amount = dictionary[s[0]];
    for (auto letter : dictionary){
      if (letter.second != amount){
        return false;
      }
    }

    return true;
  }
};